src/PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.cpp

```diff
@@ -27,21 +27,20 @@ CubicCrystal::CubicCrystal(const double c_11, const double c_12,
              "must be positive and the poisson ratio smaller or equal to 0.5.");
 }
 
-tnsr::II<DataVector, 3> CubicCrystal::stress(
-    const tnsr::ii<DataVector, 3>& strain,
-    const tnsr::I<DataVector, 3>& /*x*/) const noexcept {
-  auto result = make_with_value<tnsr::II<DataVector, 3>>(strain, 0.);
+void CubicCrystal::stress(const gsl::not_null<tnsr::II<DataVector, 3>*> stress,
+                          const tnsr::ii<DataVector, 3>& strain,
+                          const tnsr::I<DataVector, 3>& /*x*/) const noexcept {
+  std::fill(stress->begin(), stress->end(), 0.);
```

Comment:
since you're making changes to improve performance, I'd suggest assigning the `(2,2) = (0,0)` outside the loop, have the loop go from 1->3.